release-notes/release-notes57.md

```diff
@@ -37,7 +37,17 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### The `spellchecker_ignore_list` option now accepts arrays of words for specific languages
```

Comment:
Please move this note to the "Accompanying Premium Plugin changes" section. The enhancements section at the top is for core functionality and open source plugins only.

Ensure the note indicates that they are enhancements, such as:
>**Spell Checker Pro** X.Y.Z provides the following enhancements:

I'll create a separate PR to improve the boilerplate in the premium plugin section